### ISSUE TITLE
ModuleNotFoundError

### ISSUE THREAD

**Issue by BlackHat7273** [akshaynagpal/w2n#60]
I tried to install requirements.txt, it gave me an error:

ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'

I ignored that and tried to use the module pycharm told gave me an error:

Traceback (most recent call last):
  File "C:/Users/BlackHat/Desktop/ProjectFiles/module1.py", line 1, in <module>  from word2number import w2n
ModuleNotFoundError: No module named 'word2number'

**Comment by bastie** [comment]
is w2n installed?
check it with 
### pip list    
or
### pip3 list

**Comment by BlackHat7273** [comment]
Yes it is installed, and it works fine on visual studio.However, on pycharm it doesn't knowing that all other modules works just fine on pycharm

Sent from Yahoo Mail on Android 
 
  On Thu, Feb 18, 2021 at 9:39 PM, Sebastian Ritter<notifications@github.com> wrote:   


is w2n installed?
check it with

pip list

or

pip3 list
